src/main.rs

```diff
@@ -23,8 +23,10 @@ use gtk::{gio, glib};
 use once_cell::sync::OnceCell;
 use std::path::PathBuf;
 use std::str::FromStr;
+use temp_dir::TempDir;
 
 pub(crate) static APPLICATION_OPTS: OnceCell<ApplicationOptions> = OnceCell::new();
+pub(crate) static TEMP_DIR: OnceCell<PathBuf> = OnceCell::new();
```

Comment:
This can be turned into a `Lazy<Option<PathBuf>>` like we had with `RUNTIME` back then.

Reply:
The problem is that we still need to keep the `TempDir` in scope, otherwise the directory is deleted on `Drop`.

Reply:
But why should it be dropped? As I understand, the value inside lazy has a static lifetime.

Reply:
Initializing a Lazy<Option<PathBuf>> means that we create a TempDir and then we take its path, so TempDir will be dropped after the initialization.

Reply:
Also, if instead we stored the TempDir inside the static variable, the value would never be dropped.

Reply:
Ahh, I see the problem now. Did not consider the cleanup.